docs/workforce-management-framework/topics/con-raincatcher-core-getting-started.adoc

```diff
@@ -0,0 +1,143 @@
+[[id='raincatcher-core-getting-started-{chapter}']]
+= RainCatcher-core Getting Started
+
+Getting started with the Feedhenry RainCatcher Core Repository
```

Comment:
This repo is to be the single source of truth. So copy in appropriate info into this repo. We wil encounter grey areas, for example, when we need to decide to point to keyclock docs, or to other RH documentation, (data sync)...we will have to be smart when faced with this situation.